src/Relation/Binary/Partial.agda

```diff
@@ -0,0 +1,58 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of homogeneous partial equivalence relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.Partial where
+
+open import Level
+open import Relation.Binary using (Setoid)
+open import Relation.Nullary using ( ¬_)
+
+------------------------------------------------------------------------
+-- Simple properties and equivalence relations
+
+open import Relation.Binary.Core public
+
+------------------------------------------------------------------------
+-- Equivalence relations
+
+record IsPartialEquivalence {a ℓ} {A : Set a} (_≈_ : Rel A ℓ) : Set (a ⊔ ℓ) where
```

Comment:
I might be tempted to extract the `{a ℓ} {A : Set a}` out into a private variable declaration at the top of the file.